src/api/scopes/controllers/get-scopes-for-user.js

```diff
@@ -30,6 +37,7 @@ const getScopesForUserController = {
         .filter(Boolean)
     )
 
+    scopes.push(...userScopes)
```

Comment:
this can simply be `scopes.push(...userScopes, ...teamScopes)`

Reply:
I did think that, but java put me off (as it doesnt allow 2 sets of varargs)